blocks/test/fixtures/core__more.html

```diff
@@ -0,0 +1 @@
+<!-- wp:core/more {"text": "Continue reading", "noTeaser": true} /-->
```

Comment:
Right! this was generating using `GENERATE_MISSING_FIXTURES=y npm run test-unit` maybe we need to change something in the command line